services/media/service/src/ingest/handlers/video-succeeded-handler.ts

```diff
@@ -72,6 +72,19 @@ export abstract class VideoSucceededHandler<
     ).run(ownerClient);
   }
 
+  public override mapError(error: unknown): Error {
+    const fallbackMessage =
+      this.messagingSettings.messageType ===
+      VideoServiceMultiTenantMessagingSettings.EnsureVideoExistsAlreadyExisted
```

Comment:
Here and in the other similar handler: as you are anyways using abstract classes I would suggest to not do such if statement here but call an abstract method/property where the child class would then provide that text.